docs/IoT.js-API-GPIO.md

```diff
@@ -33,7 +42,7 @@ After this function any other GPIO function call except `initialize()` will be f
 'release' event will be emitted after GPIO device is released.
 
 
-#### gpio.setPin(pinNumber, direction[, mode][, callback])
+#### gpio.open(pinNumber, direction[, mode][, callback])
```

Comment:
I think ,in this section, we need to explain why ``open`` API is get △ in Platform Support section.